README.md

```diff
@@ -98,6 +98,28 @@ To be expanded...but in short:
  6. Play a ROSBag on ROS computer
  7. View meshes in Unity
 
+### Create Mesh in ArUco marker coordinate system
+Here are relevant files:
+ 1. src\isaacs_mapping\launch\isaacs_mapping.launch
+ 2. src\isaacs_mapping\launch\zed_voxblox.launch
+ 3. src\isaacs_mapping\src\process_aruco.py
+
+Zed camera needs to publish the following topics:
+ 1. Point Cloud: '/zed2/zed_node/mapping/fused_cloud'
+ 2. Image: '/zed2/zed_node/rgb/image_rect_color' or '/zed2/zed_node/rgb/image_rect_color/compressed'
+
+The script subscribes to topics published by the ZED camera. It uses the images to detect ArUco marker, and converts point clouds to the marker coordinate systems by changeing their positions and publish new point clouds to a new topic. The voxblox has been set up to generate mesh based on this new point cloud topic. By default the script is receiving non-compressed images. If you want to use compressed images instead, at the end of process_aruco.py use PointCloudCamToMarkerConverter(image_is_compressed = True) to create the converter object.
+
+Following these steps to run the script:
+ 1. Get IP address of ROS computer
```

Comment:
Should we be more specific about "ROS computer"? Something like the onboard computer?

Reply:
we were using ROS computer in previous parts in the readme, so I used this word as well to be consistent.